### ISSUE TITLE
have original columns appended to the matching result

### ISSUE THREAD

**Issue by XiaosenChen** [TresAmigosSD/SMV#278]
current output: 
id|_id|Full_Name_Match
---|---|---
2|1|true
1|3|false

desired output:
id|_id|Full_Name_Match|Full_Name|_Full_Name
---|---|---|---|---
2|1|true|Fred Flintstone|Fred Flintstone
1|3|false|George Jetson|Georje Jetson

**Comment by ninjapapa** [comment]
May consider to add an optional parameter to the `doMatch` method to allow keeping the input columns. The signature of `doMatch` could be
```scala
def doMatch(df1: DataFrame, df2: DataFrame, keepOriginalCols: Boolean = false)
```

**Comment by ninjapapa** [comment]
addressed in #483
